server/queries/readSurveys.js

```diff
@@ -0,0 +1,13 @@
+const { getDb } = require('../databaseConnection');
+const readUsers = async () => {
+  try {
+    const db = getDb();
+    const surveys = db.collection('Surveys');
+    const result = await surveys.find({}).toArray();
```

Comment:
I'm not convinced you need the {} inside find().

Reply:
true. done